scipy/optimize/_basinhopping.py

```diff
@@ -422,8 +422,7 @@ def basinhopping(func, x0, niter=100, T=1.0, stepsize=0.5,
     niter_success : integer, optional
         Stop the run if the global minimum candidate remains the same for this
         number of iterations.
-    seed : {None, int, `numpy.random.Generator`,
-            `numpy.random.RandomState`}, optional
+    seed : {None, int, `numpy.random.Generator`, `numpy.random.RandomState`}, optional
```

Comment:
This one is my fault and is present elsewhere? Would you mind taking care of these in this PR? (If not no problem OC)

Reply:
Actually it would be too long. This is why I initially do that, but the formatting is incorrect as you identified. Do you have a suggestion?

Reply:
Yeah, I don't have any other suggestions. I don't know if just "Generator" and "randomstate" would be resolved.

Reply:
Are you at SciPy ? Should we mark that as something to look at during the sprints ?
I've started my travels so can probably look at that in the plane tomorrow.

Reply:
Yes I am. I will Slack you 😃

Reply:
Ark we forgot about this one. I was having a look at Pandas and apparently they don't split lines. So maybe there is no easy way to do this otherwise.

Reply:
Agreed, if something doesn't fit in 80 char it doesn't fit (goes for html links too often). Leaving out a correct thing like `, optional` is definitely not desired.